src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/MålgruppeVilkår.tsx

```diff
@@ -20,8 +19,8 @@ const Container = styled.div`
 const MålgruppeVilkår: React.FC<{
     målgruppeForm: EndreMålgruppeForm;
     oppdaterDelvilkår: (key: keyof DelvilkårMålgruppe, vurdering: Vurdering) => void;
-    felterSomKanEndres: FelterSomKanEndresIPerioden;
-}> = ({ målgruppeForm, oppdaterDelvilkår, felterSomKanEndres }) => {
+    readOnly: boolean;
+}> = ({ målgruppeForm, oppdaterDelvilkår, readOnly }) => {
```

Comment:
Endret denne så den var lik som `AktivitetVilkår`, men man kunne også gått for at denne tok inn "alle er låst"

Reply:
Jeg liker `readOnly` på komponentene som er ett enkelt inputfelt, men "alle er låst" på komponenter som inneholder flere felter